automation/aws/aws_rbd_from_tag/aws_rbd_from_tag.pt

```diff
@@ -7,7 +7,7 @@ severity "low"
 category "Cost"
 default_frequency "daily"
 info(
-  version: "2.2.1",
+  version: "2.3.1",
```

Comment:
Should the version be `"2.3.0"`? (a question to all templates updated as part of this PR)